src/dbus_api/blockdev.rs

```diff
@@ -31,10 +32,10 @@ use super::util::get_uuid;
 use super::util::ok_message_items;
 
 
-pub fn create_dbus_blockdev<'a>(dbus_context: &DbusContext,
-                                parent: dbus::Path<'static>,
-                                uuid: Uuid)
-                                -> dbus::Path<'a> {
+pub fn create_dbus_blockdev(dbus_context: &DbusContext,
```

Comment:
Rather than pulling the code out of ```create_dbus_blockdev``` I think you're better off putting more in. Is it possible to add a parameter that refers to the list that needs to be updated here?